lumber-generate.js

```diff
@@ -9,7 +9,8 @@ const logger = require('./services/logger');
 
 program
   .description('Generate a backend application with an ORM/ODM configured.')
-  .option('-c, --connection-url <connectionUrl>', 'Enter the database credentials with a connection URL')
+  .option('-c, --connection-url <connectionUrl>', 'Enter the database credentials with a connection URL.')
+  .option('--schema <schema>', 'Enter your database schema.')
```

Comment:
It seems the version without a final dot is preferred by command help. I tried `docker --help`, `npm --help`, `node --help` and `yarn --help`